src/control/signal.c

```diff
@@ -322,15 +349,18 @@ void dt_control_signal_raise(const dt_control_signal_t *ctlsig, dt_signal_t sign
   }
 }
 
-
 void dt_control_signal_connect(const dt_control_signal_t *ctlsig, dt_signal_t signal, GCallback cb,
                                gpointer user_data)
 {
+  dt_print(DT_DEBUG_SIGNAL, "[signal] connect: %s\n", _signal_description[signal].name);
+  _print_trace("connect");
   g_signal_connect(G_OBJECT(ctlsig->sink), _signal_description[signal].name, G_CALLBACK(cb), user_data);
 }
 
 void dt_control_signal_disconnect(const struct dt_control_signal_t *ctlsig, GCallback cb, gpointer user_data)
 {
+  dt_print(DT_DEBUG_SIGNAL, "[signal] disconnect\n");
```

Comment:
not sure of this one : as it doesn't show the signal name, it will be useless imho and just spam the terminal